src/SamplesApp/UITests.Shared/ValueConverters/BoolNegationValueConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml.Data;
+
+namespace UITests.Shared.ValueConverters
+{
+	public class BoolNegationValueConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, string language)
+		{
+			if ( value is bool boolValue)
+			{
+				return !boolValue;
+			}
+			return false;
+		}
+
+		public object ConvertBack(object value, Type targetType, object parameter, string language)
+		{
+			throw new NotImplementedException();
```

Comment:
Per nventive's convention:
`NotImplementedException` means your job is not finished
`NotSupportedException` means it's not expected to be implemented.

Reply:
Will do